lib/mongoid/extensions/raw_value.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Wrapper class used when a value cannot be casted in evolve method.
+module Mongoid
+  def RawValue(*args)
```

Comment:
I know it's pretty self-evident, but a comment to document this method would be clearer, I think.

Reply:
Done!